backend/controllers/taskController.js

```diff
@@ -15,4 +19,101 @@ const createTask = asyncHandler(async (req, res) => {
   }
 });
 
-module.exports = { createTask };
+const getTasks = asyncHandler(async (req, res) => {
+  // TESTED
+  // console.log("in GetTasks");
+  const userId = req.body.user_id;
+  console.log(userId);
+  const tasks = await Task.find({ user_id: userId });
+
+  // console.log("TASKS:");
+  // console.log(tasks);
+  res.status(200).json(tasks);
+});
+
+const getTasksById = asyncHandler(async (req, res) => {
+  // Task id Parameter
+  console.log("In getTasksById");
+  const id = req.params.id;
```

Comment:
Minor comment: We can rename this to `taskId` rather than just `id` such that it's more descriptive

Reply:
Fixed: Changes are reflected in recent commits